.github/workflows/taintyper-ci.yml

```
@@ -0,0 +1,498 @@
+      run: npm run build
+
+    - name: ⚡ Large Scale Performance Test
+      run: |
+        echo "⚡ 大規模性能回帰テスト開始"
+        npm test -- LargeScalePerformance.test.ts
+        echo "✅ Performance regression test completed"
+      env:
+        NODE_OPTIONS: '--max-old-space-size=6144'
+
+    - name: 📈 Performance Trend Analysis
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2067:error:4:60: Missing ';' or + terminating -exec. You can't use |/||/&&, and ';' has to be a separate, quoted argument

(shellcheck)